index.d.ts

```diff
@@ -39,11 +39,10 @@ declare module 'i18next' {
 type ObjectOrNever = TypeOptions['allowObjectInHTMLChildren'] extends true
   ? Record<string, unknown>
   : never;
-type ReactI18NextChild = React.ReactNode | ObjectOrNever;
 
 declare module 'react' {
   interface HTMLAttributes<T> {
-    children?: ReactI18NextChild | Iterable<ReactI18NextChild>;
```

Comment:
I don't remember why and when we did that, but it doesn't make much sense, since `ReactNode` is already `Iterable`

Reply:
originally introduced with #1531 by @gshokanov

Reply:
//cc @lexanth @cravend @Igosuki @gabriel-dehan